homeassistant/components/squeezebox/services.yaml

```diff
@@ -30,3 +30,41 @@ call_query:
       advanced: true
       selector:
         object:
+search:
+  target:
+    entity:
+      integration: squeezebox
+      domain: media_player
+  fields:
+    command:
+      required: true
+      selector:
+        select:
+          multiple: false
+          options:
+            - label: Albums
+              value: albums
+            - label: Artists
+              value: artists
+            - label: Tracks
+              value: tracks
+            - label: Playlists
+              value: playlists
+            - label: Genres
+              value: genres
+            - label: Favorites
+              value: favorites
+            - label: Players
+              value: players
```

Comment:
Rather create this a proper selector with translatable names

Reply:
Sorry, could you explain what you mean by a "proper" selector?  Do you just mean adding translation strings?  Much of the rest of the strings are currently fixed english only, although I know they should also likely be translation objects

Reply:
I was in the gym so I had no links ready. Check the service action for `bring`

Reply:
Thanks - will update it

Reply:
done